content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,148 @@
+overhead, low CPU (~1%), and memory usage in production environments.
+
+The Elastic profiling agent facilitates identifying non-optimal code paths,
+uncovering "unknown unknowns", and provides comprehensive visibility into the
+runtime behavior of all applications. Elastic’s continuous profiling agent
+provides support for a wide range of runtimes and languages, such as:
+
+- C/C++
+- Rust
+- Zig
+- Go
+- Java
+- Python
+- Ruby
+- PHP
+- Node.js / V8
+- Perl
+- .NET
+
+Additionally, organizations can combine profiling data with resource information
+to derive an estimate for the carbon footprints of services. This helps
+organizations meet their sustainability objectives by minimizing computational
+wastage, ensuring seamless alignment with their strategic ESG goals.
+
+## Benefits to OpenTelemetry
+
+This contribution not only boosts the standardization of continuous profiling
+for observability but also accelerates its adoption as a key signal in
+OpenTelemetry. Customers benefit from a vendor-agnostic method of collecting
+profiling data correlating it with existing signals, like tracing, metrics, and
+logs, opening new potential for observability insights and a more efficient
+troubleshooting experience.
+
+### User Benefits of OpenTelemetry Profiling
+
+OpenTelemetry-based continuous profiling unlocks the following possibilities for
+users:
+
+- Continuous profiling data compliments the existing signals (traces, metrics
+  and logs) by providing detailed, code-level insights on the services'
+  behavior.
+
+- Seamless correlation with other OpenTelemetry signals such as traces,
+  increasing fidelity and investigatory depth.
+
+- Minimize environmental impact: energy consumption associated with computing is
+  a growing concern (source:
+  [MIT Energy Initiative](https://energy.mit.edu/news/energy-efficient-computing/)).
+  More efficient code translates to lower energy consumption, contributing to a
+  reduction in carbon (CO2) footprint. Combining profiling data with
+  OpenTelemetry's resource information (i.e. resource attributes) allows to
+  derive insights into the services' carbon footprint.
+
+- Through a detailed breakdown of services' resource utilization, profiling data
+  provides actionable information on performance optimization opportunities.
+
+- Improved vendor neutrality: a vendor-agnostic eBPF-based profiling agent
+  removes the need to rely on proprietary agents to collect profiling telemetry.
+
+With these benefits, SREs, developers, and DevOps, can now manage the overall
+application’s efficiency on the cloud while ensuring their engineering teams
+optimize it.
+
+As the next step, Elastic and the OpenTelemetry profiling SIG will jointly work
+on integrating the donated agent into OpenTelemetry's components ecosystem. We
+look forward to providing a fully integrated and usable version of the new
+OpenTelemetry eBPF profiling agent to the users, soon.
```

Comment:
Great closing statement, one _nit_:

"Elastic and the OpenTelemetry profiling SIG" makes it appear that Elastic stands outside of that SIG. Maybe we can rephrase that a little bit?